spec/chewy/multi_search_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'spec_helper'
+require 'chewy/multi_search'
+
+describe Chewy::MultiSearch do
+  before { Chewy.massacre }
+
+  before do
+    stub_model(:city)
+    stub_model(:country)
+
+    stub_index(:places) do
+      define_type City do
+        field :name, **KEYWORD_FIELD
```

Comment:
Better replace this constant with
```suggestion
        field :name, type: 'keyword'
```

Reply:
Done!